recipes/libplist/all/conanfile.py

```diff
@@ -0,0 +1,55 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibplistConan(ConanFile):
+    name = "libplist"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/libimobiledevice-win32/libplist"
+    description = "A library to handle Apple Property List format in binary or XML"
+    topics = ["conan", "plist", "apple"]
+    license = "LGPL-2.1"
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {'shared': False,
+                       'fPIC': True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "libplist"
+
+    def configure(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "libplist-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
```

Comment:
I know it's a pain on Windows, but this project provides autotools files. Do your properly emulate what happen in upstream autoconf.ac and Makefile.am?

Reply:
Autotools is hard to cross-platform compilation. so I do not try it. This `CMakeLists.txt` is uesd to compile libplist and it work well, so I just take it and use it.